examples/Nat/ITransportState.cs

```diff
@@ -0,0 +1,60 @@
+/*
+Pax : tool support for prototyping packet processors
+Jonny Shipton, Cambridge University Computer Lab, July 2016
+
+Use of this source code is governed by the Apache 2.0 license; see LICENSE.
+*/
+
+using System;
+using PacketDotNet;
+
+namespace Pax.Examples.Nat
+{
+  /// <summary>
+  /// Provides a typed interface for the current state of a Transport-layer protocol.
+  /// </summary>
+  /// <typeparam name="T">The type of packet</typeparam>
+  public interface ITransportState<T> where T : Packet
+  {
+    /// <summary>
+    /// Gets a boolean value determining if the connection can be removed before the timeout elapses.
```

Comment:
I think it needs to made clearer what role this can play. I think it's meant to capture the situation, as one might find in TCP, where a connection has been closed (via FIN exchange) and therefore we don't need to wait until the records expiry for the GC to remove the entry. But then even in TCP, doesn't the "TIME-WAIT" timer keep a connection from being considered "closed" until the timer expires? It's not clear what "timeout" refers to in the comment.